scripts/pylint_extensions.py

```diff
@@ -1385,7 +1385,11 @@ def process_module(self, node):
 
 
 class SpaceBelowFileOverviewChecker(checkers.BaseChecker):
-    """Checks if there is an empty line between file overview and imports."""
+    """Checks if there is an empty line between file overview and imports.
+       Note: The check assumes that all files have a file overview, this
```

Comment:
Indentation is wrong. Deindent this and the following lines by 3.

Also change the comma to a period and start a new sentence with "This".

Reply:
done